x-pack/plugins/apm/server/routes/backends/get_top_backend_spans.ts

```diff
@@ -78,6 +82,18 @@ export async function getTopBackendSpans({
               ...kqlQuery(kuery),
               ...termQuery(SPAN_DESTINATION_SERVICE_RESOURCE, backendName),
               ...termQuery(SPAN_NAME, spanName),
+              ...((sampleRangeFrom ?? 0) >= 0 && (sampleRangeTo ?? 0) > 0
```

Comment:
Yes, but same applies here: I prefer to check against >= 0 to make sure I'm actually comparing numbers to numbers instead of accidentally casting something to a number.